posts/inside-rust/2023-09-19-crates-io-usage-policy-rfc.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: crates.io Policy Update RFC
+author: Tobias Bieniek
+team: the crates.io team <https://www.rust-lang.org/governance/teams/crates-io>
+---
+
+Around the end of July the crates.io team opened an [RFC](https://github.com/rust-lang/rfcs/pull/3463) to update the current crates.io usage policies. This update addresses operational concerns that have arisen since the last significant update in 2017, particularly related to name squatting and spam. This has caused considerable discussion, and most of the suggested improvements have since been integrated into the proposal.
+
+At the last team meeting the crates.io team decided to move the RFC forward and start the **final comment period** process.
+
+We have been made aware by a couple of community members though that the RFC might not have been visible enough in the Rust community. We hope that this blog post changes that.
```

Comment:
I think it should be on main blog, not inside rust, if you're looking specifically to talk to community. But not a strong opinion and probably doesn't matter that much.

Reply:
I also think it should be the main blog.

Reply:
> I think it should be on main blog

after discussion in the crates.io team meeting we decided to move it to the main blog